core/templates/services/math-interactions.service.ts

```diff
@@ -159,6 +159,16 @@ export class MathInteractionsService {
           variable, separatedVariables);
       }
     }
+    // Inserting multiplication signs before functions. For eg. 5sqrt(x) should
```

Comment:
Does this work for cases like `sqrt(5)log(2)` also?

Reply:
Yes, I've added test cases for the same here:
https://github.com/oppia/oppia/blob/eb3b6056c8133d1e88855b8ef186a217d2a0c041/core/templates/services/math-interactions.service.spec.ts#L270-L275

Reply:
Ah ok!